acceptance-tests/xfcc_test.go

```diff
@@ -293,3 +351,32 @@ func parseXFCCHeader(header string) string {
 
 	return certPEM.String()
 }
+
+func parseXFCCHeaderToCert(header string) *x509.Certificate {
+	derEncodedCert, err := base64.StdEncoding.DecodeString(header)
+	Expect(err).NotTo(HaveOccurred())
+
+	cert, err := x509.ParseCertificate(derEncodedCert)
+	Expect(err).NotTo(HaveOccurred())
+
+	return cert
+}
+
+func checkXFCCHeaders(recordedXFCCHeader string, recordedHeaders http.Header) {
+	cert := parseXFCCHeaderToCert(recordedXFCCHeader)
+	Expect(recordedHeaders.Get("X-Ssl-Client")).To(Equal("1"))
+	Expect(recordedHeaders.Get("X-Ssl-Client-Verify")).To(Equal("0"))
+	Expect(recordedHeaders.Get("X-Ssl-Client-Subject-Dn")).To(ContainSubstring(cert.Subject.Country[0]))
+	Expect(recordedHeaders.Get("X-Ssl-Client-Subject-Dn")).To(ContainSubstring(cert.Subject.Organization[0]))
+	Expect(recordedHeaders.Get("X-Ssl-Client-Subject-Dn")).To(ContainSubstring(cert.Subject.CommonName))
+	Expect(recordedHeaders.Get("X-Ssl-Client-Subject-Cn")).To(Equal(cert.Subject.CommonName))
+	Expect(recordedHeaders.Get("X-Ssl-Client-Issuer-Dn")).To(ContainSubstring(cert.Issuer.Country[0]))
+	Expect(recordedHeaders.Get("X-Ssl-Client-Issuer-Dn")).To(ContainSubstring(cert.Issuer.Organization[0]))
+	Expect(recordedHeaders.Get("X-Ssl-Client-Issuer-Dn")).To(ContainSubstring(cert.Issuer.CommonName))
+	notBeforeString := fmt.Sprintf("%02d%02d%02d%02d%02d%02dZ", cert.NotBefore.Year()-2000,
+		cert.NotBefore.Month(), cert.NotBefore.Day(), cert.NotBefore.Hour(), cert.NotBefore.Minute(), cert.NotBefore.Second())
+	Expect(recordedHeaders.Get("X-Ssl-Client-Notbefore")).To(Equal(notBeforeString))
+	notAfterString := fmt.Sprintf("%02d%02d%02d%02d%02d%02dZ", cert.NotAfter.Year()-2000,
+		cert.NotAfter.Month(), cert.NotAfter.Day(), cert.NotAfter.Hour(), cert.NotAfter.Minute(), cert.NotAfter.Second())
+	Expect(recordedHeaders.Get("X-Ssl-Client-Notafter")).To(Equal(notAfterString))
+}
```

Comment:
VSCode: `"files.insertFinalNewline": true` :)

Otherwise can cause some problems with unix tools

Reply:
I'm using Goland but there should be a similar feature.

Reply:
seems go fmt does that already. I've turned it on in Goland and it added the newline.